src/schema/ecommerce/types/initial_offer_input_type.ts

```diff
@@ -1,19 +1,16 @@
-import {
-  GraphQLInputObjectType,
-  GraphQLNonNull,
-  GraphQLInt,
-  GraphQLID,
-} from "graphql"
+import { GraphQLInputObjectType, GraphQLNonNull, GraphQLID } from "graphql"
+import { MoneyInput } from "schema/fields/money"
+
 export const InitialOfferInputType = new GraphQLInputObjectType({
   name: "InitialOfferOrderInput",
   fields: {
     orderId: {
       type: new GraphQLNonNull(GraphQLID),
       description: "ID of order",
     },
-    amountCents: {
-      type: new GraphQLNonNull(GraphQLInt),
-      description: "Offer amount in cents",
+    offerAmount: {
+      type: MoneyInput,
+      description: "Offer amount",
```

Comment:
Maybe to address @joeyAghion 's point above, we talked about making the description here explicit.

Reply:
yes makes sense to add it here! although im really having hard time to find out how to make it generically explicit, `Offer amount in dollars`? `offer amount in major`?

Reply:
updated description here and in `MoneyField`